### PR TITLE
stop sync with checking status

### PR DESCRIPTION
**Pull request by informalict**

### CODE REVIEW

File: release_tester/arangodb/starter/deployments/dc2dc.py [hunk 1]
```diff
@@ -208,6 +208,29 @@ def _is_higher_sync_version(self, min_v1_version, min_v2_version):
 
         return is_higher_version(self.sync_version, min_v2_version)
 
+    def _get_sync_status(self, target_cluster=True):
+        """
+            Check replication status of the cluster.
+        """
+        cluster_instance = self.cluster1['instance']
```
Comment (dothebart) [inline release_tester/arangodb/starter/deployments/dc2dc.py:243]: please use 
```
cluster = self.cluster1 if not target_cluster else self.cluster
```
and work with sub-references of cluster below.

File: release_tester/arangodb/starter/deployments/dc2dc.py [hunk 2]
```diff
@@ -241,16 +264,23 @@ def _stop_sync(self, timeout=120):
...
+            self.state += "\n" + output
+            raise Exception("failed to stop the synchronization")
+
+        count = 0
+        status_source = ""
+        status_target = ""
+        while count < 30:
+            status_source = self._get_sync_status(False)
+            status_target = self._get_sync_status(True)
+            if status_source == "inactive" and status_target == "inactive":
```
Comment (dothebart) [inline release_tester/arangodb/starter/deployments/dc2dc.py:309]: here we should have:
```
            status_source = self._get_sync_status(self.cluster1)
            status_target = self._get_sync_status(self.cluster2)
```

Comment (informalict) [reply release_tester/arangodb/starter/deployments/dc2dc.py:282]: changed

File: release_tester/arangodb/starter/deployments/dc2dc.py (hunk 1 above)
Comment (informalict) [reply release_tester/arangodb/starter/deployments/dc2dc.py]: changed

**Review by dothebart — APPROVED** [review]

**Pull request merged**